README.md

```diff
@@ -1 +1,84 @@
+- Given a trip object, you should be able to retrieve the associated rider instance through the rider ID
+- You should be able to find all trip instances for a given driver ID
+- You should be able to find all trip instances for a given rider ID
+- You should be able to retrieve all trips from the CSV file
+
+## Getting Started
+This is a level 3, individual project.
+
+We will use the same project structure we used for the previous project. Classes should be in files in the `lib` folder, and tests should be in files in the `specs` folder. You should utilize a spec helper file. You will run tests by executing the `rake` command, as configured in a Rakefile. You should have **95% code coverage** using simplecov.
+
+The `support` folder contains CSV files which will drive your system design. Each CSV corresponds to a different type of object _as well as_ creating a relationship between different objects.
+
+## Baseline
+### Discussion
+#### Phase 1
+Let's start by **individually** creating the list of nouns and verbs that you would utilize to describe this problem.
+
+After a few minutes, then you'll get together with your seat squad to discuss the nouns and verbs you came up with.
+
+#### Phase 2
+Use your list of nouns to begin thinking about what classes you will want to use. Use your list of verbs associated with nouns to consider what methods you need to put in which classes.
+
+After a few minutes, then you'll get together with your seat squad to discuss the classes and methods you came up with.
+
+#### Phase 3
+Create a diagram that describes how each of these classes and methods (messages) will interact with one another as well as the CSV files.
+
+### Setup
+1. You will fork this repository in GitHub
+1. You will git clone the repository to your computer
+1. Create/copy a rakefile to run your tests
+1. Create a spec helper file to load your classes and start up simple coverage
+1. Create a test to check the instantiation of one of your object types (**RED**)
+1. Create the class for the object tested in the step above (**GREEN**)
+1. Use git add, commit and push commands to push your initial code to GitHub
+
+### Process
+You should use the following process as much as possible:  
+
+1. Write pseudocode
+1. Write test(s)
+1. Write code
+
+Since this is the **last** Ruby-only project we are doing, and a level 3, the requirements are larger than previous projects. It is possible you will not be able to complete all requirements, but if you use the process above, you will still be able to share with your instructors the process and the design you have created.
+
+### Optional Requirements
+- The first optional should always be looking for improvements within the existing code you have already written. Are there any additional edge cases you might be missing in your tests? Any opportunity to use an enumerable method where you're using an each?
```

Comment:
If this should always be the first optional, are we planning to update future project READMEs to match?